### DISCUSSION TITLE
Possible to assign both IPv4 and IPv6 to an initiator?

### DISCUSSION THREAD

**Discussion by hgl** [strongswan/strongswan#211] [Q&A] [answered]
I set `pools.<name>.addrs` to something like `8.8.8.8,fec0::/64`, but the initiator is only assigned an IPv4 address upon connection. The initiator is a Mac, I wonder if it's a limitation of the Mac IKEv2 client or a configuration error or charon doesn't support dual stack?

**Comment by tobiasbrunner** [comment, marked as answer]
You have define separate pools for different address families or even multiple subnets/ranges of the same family.

**Reply by hgl** [reply]
Works like a charm, thanks.
